src/components/ErrorBox.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { APIError } from "../data/types"
+
+interface ErrorBoxParams {
+  title: string
+  retry: () => void
+  error: APIError
```

Comment:
This should be of type `string`, the user of `ErrorBox` can unpack the `APIError`.